### PR TITLE
tomcat_mgr_deploy tomcat_mgr_upload docs

### PR DESCRIPTION
**Pull request by h00die**
Adds docs for `exploit/multi/http/tomcat_mgr_deploy ` and `exploit/multi/http/tomcat_mgr_upload `

Of note, I always deployed tomcat to 808# where # was the version number, so Tomcat6 was 8086, 7 was 8087 etc.  Should make docs easier to ready through.

## Verification

- [x] Start `msfconsole`
- [x] `use exploit/multi/http/tomcat_mgr_deploy`
- [x] `info -d`
- [x] **Verify** spellings, flow makes sense, etc.  
- [x] `use exploit/multi/http/tomcat_mgr_upload`
- [x] `info -d`
- [x] **Verify** spellings, flow makes sense, etc.

### CODE REVIEW

File: documentation/modules/exploit/multi/http/tomcat_mgr_deploy.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+    Computer     : 192.168.2.118
+    OS           : Ubuntu 16.04 (Linux 4.4.0-59-generic)
+    Architecture : x64
+    Meterpreter  : x86/linux
+    ```
+
+#### gui Interface
+The permission `manager-gui`, is required for this exploitation, and the `PATH` should be `/manager/html`.  However, my attempts were unsuccessful, also wasn't able to figure out where the `check` command's `serverinfo` URL was located.  [Manual exploitation is possible](#manual-exploitation)
+
+# Manual Exploitation
```
Comment (h00die) [inline documentation/modules/exploit/multi/http/tomcat_mgr_deploy.md]: This entire section is actually from `tomcat_mgr_upload` which i'll be working on next.

**Comment by h00die** [comment]
since the docs are VERY similar, put 2 docs in this one request.

**Comment by busterb** [comment]
This looks good. I had to think a bit why we have these two modules to begin with, but I can see them serving different purposes perhaps (depending on what verbs are allowed through an ALB for instance.)

**Comment by busterb** [comment]
## Release Notes

This adds documentation for the tomcat_mgr_deploy and tomcat_mgr_upload modules.

**Comment by h00die** [comment]
I tried to answer that in the docs as well since I had the same question

**Pull request merged**